HubSpot.NET/Api/Engagement/EngagementListRequestOptions.cs

```diff
@@ -23,11 +23,9 @@ public int Limit
             get => _limit;
             set
             {
-                if (value < 1 || value > 100)
-                {
-                    throw new ArgumentException(
-                        $"Number of contacts to return must be a positive integer greater than 0 - you provided {value}");
-                }
+                if (value < 1 || value > 100)                
+                    throw new ArgumentException($"Number of contacts to return must be a positive ingeteger greater than 0 - you provided {value}");              
```

Comment:
Minor type - no need to fix here but maybe worth tidying up at some point.

Reply:
Ah yes

Reply:
Fixed it